### DISCUSSION TITLE
行政院要設「數位發展部」!

### DISCUSSION THREAD

**Discussion by Joan0518** [1101-GS4719-C/Python-Community-GS4719-C#229] [General]
立法院院會在12/28三讀通過 「數位發展部組織法」，行政院將新設數位發展部，掌管通訊、資訊、資通安全、網路及傳播五大領域，並設立資安署和數位產業發展署，強化資安政策，促進數位經濟發展及數位科技應用的政策規劃。
「數位發展部」將整體規劃國家數位發展政策，統籌基礎建設、環境整備及資源運用業務，並設置次級機關數位產業署、資通安全署，以達成國家數位轉型的目的。

台灣將一步一步走入數位化的時代，也相信有助於未來推出央行數位貨幣。
話說部長會是唐鳳嗎:satisfied:
![數位發展部](https://user-images.githubusercontent.com/90574174/147631766-947e7b00-1d99-40f7-b450-5cf42cbff61d.jpg)
[https://technews.tw/2021/12/28/digital-development-department/](url)

**Comment by HSU-YU-JEN** [comment]
政府機關終於重視這類科技發展了!!!

**Comment by JAIMEHUNG** [comment]
希望能在注重科技發展的同時，能夠在法律制度上也跟進～

**Comment by 109802503gloria** [comment]
透過相關部門的整合，感覺可以在行政效率上快很多
